data/mainnet/tokens.js

```diff
@@ -45,6 +45,48 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['FARM', 'WETH'],
   },
+  idleTranche_WSTETH: {
+    isNew: true,
+    category: VAULT_CATEGORIES_IDS.IDLE,
+    chain: CHAINS_ID.ETH_MAINNET,
+    logoUrl: './icons/idletranche-wsteth.png',
+    apyIconUrls: ['./icons/idle.png', './icons/ldo.png'],
+    apyTokenSymbols: ['IDLE', 'LDO'],
+    displayName: 'wstETH',
+    subLabel: 'Idle Finance Tranche',
+    tokenAddress: addresses.V2.idleTranche_WSTETH.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.V2.idleTranche_WSTETH.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.COINGECKO_CONTRACT,
+      params: [addresses.WSTETH],
+    },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.IDLE_FINANCE_TRANCHE,
+        params: [
+          '0x34dCd573C5dE4672C8248cd12A99f875Ca112Ad8',
```

Comment:
Could we add these addresses to `addresses.json`, together with `newVault` etc? Makes it more readable